sklearn/metrics/tests/test_classification.py

```diff
@@ -2537,7 +2564,7 @@ def test_log_loss_pandas_input():
     y_pr = np.array([[0.2, 0.7], [0.6, 0.5], [0.4, 0.1], [0.7, 0.2]])
     types = [(MockDataFrame, MockDataFrame)]
     try:
-        from pandas import Series, DataFrame
+        from pandas import DataFrame, Series
```

Comment:
This can be reverted so a reviewer can focus on the changes that have to zero division = nan